integrations/spring.hbs.md

```diff
@@ -3,13 +3,30 @@
 This topic describes how to migrate Spring applications from Tanzu Application Service or
 Azure Spring Apps to Tanzu Application Platform.
 
-## <a id="prereqs"></a> Prerequisite
+## <a id="spring-cloud-config"></a> Migrating from Spring Cloud Config
 
-If your applications are currently using Spring Cloud Configuration Service or
-Spring Cloud Service Registry,
-[install the Application Configuration Service](../application-configuration-service/install-app-config-service.hbs.md)
+If your applications are currently using
+[Spring Cloud Config](https://spring.io/projects/spring-cloud-config), refer to
+installing Tanzu Application Platform compatible option
+[Application Configuration Service](../application-configuration-service/install-app-config-service.hbs.md)
 optional package.
 
+## <a id="spring-cloud-gateway"></a> Using Spring Cloud Gateway for Kubernetes
+
+Spring Cloud Gateway is a popular project library for creating an API Gateway built on top of the
+Spring ecosystem. The OSS library is a foundational component of our Spring Cloud Gateway for
+Kubernetes and Spring Cloud Gateway for VMware Tanzu Application Service commercial offerings along
+with commercial-only capabilities and platform integrated operator experiences. The OSS and
+commercial offerings can be used as a reverse proxy with additional API Gateway functionality to
+handle request and response to upstream application services.
+
+With Spring Cloud Gateway for Kubernetes now included as part of Tanzu Application Platform
```

Comment:
This isn't a complete sentence. Maybe say "Spring Cloud Gateway for Kubernetes is now included as part of Tanzu Application Platform..." instead?

Reply:
The PR is now merged and I've already corrected this grammar, thanks